tests/pages/pages.py

```diff
@@ -1107,7 +1107,7 @@ class ManageFolderPage(BasePage):
     delete_link = (By.LINK_TEXT, "Delete this folder")
     name_input = NameInputElement()
     delete_button = (By.NAME, "delete")
-    save_button = (By.CSS_SELECTOR, "main [type=submit]")
+    save_button = (By.CSS_SELECTOR, "form button")
```

Comment:
Same as last comment, with JS on this page has 2 buttons:

<img width="368" alt="image" src="https://user-images.githubusercontent.com/87140/200561904-691708b2-9fe9-4c6a-8c79-5b7d14f07c22.png">

Same advice as before applies here:

```suggestion
    save_button = (By.CSS_SELECTOR, "form button.page-footer__button")
```